scripts/run_migrations.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-RESULT=$(scl enable rh-python38 "${APP_HOME}/manage.py check_migrations | tail -1")
+RESULT=$(python3.8 ${APP_HOME}/manage.py check_migrations | tail -1)
```

Comment:
Why this change?

Reply:
In the base ubi8/python-38 image,`python3.8` is enabled for us, so we don't need the extra `scl enable` syntax:
```
RUN source scl_source enable rh-python38 && \
    python3.8 -m venv ${APP_ROOT} && \
    chown -R 1001:0 ${APP_ROOT} && \
    fix-permissions ${APP_ROOT} -P && \
    rpm-file-permissions
```
Is this change _necessary_? Not likely, but I do think it simplifies the script just a tiny bit.